client/state/login/actions.js

```diff
@@ -156,7 +158,10 @@ export const loginUserWithTwoFactorVerificationCode = ( user_id, two_step_code,
  */
 export const sendSmsCode = ( userId, twoStepNonce ) => dispatch => {
 	dispatch( {
-		type: TWO_FACTOR_AUTHENTICATION_SEND_SMS_CODE_REQUEST
+		type: TWO_FACTOR_AUTHENTICATION_SEND_SMS_CODE_REQUEST,
+		notice: {
+			message: translate( 'Trying to send you a text message…' )
```

Comment:
This is a bit of a nitpick, but this message sounds a little pessimistic with "Trying to send" instead of just "Sending". This should pretty much never fail.